app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  before_action :set_post
+
+  def create
+    @comment = @post.comments.new(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
```

Comment:
- [ ] I would kind kindly suggest you add a method in the commentController to add a new comment as this will make your project look perfect

e.g
```ruby 
    def new
       @comment = Comment.new
    end
```

And also I would employ you to add the LikesController and input the necessary methods i.e the create method